aten/src/ATen/native/cpu/DepthwiseConvKernel.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/native/DispatchStub.h>
+
+/*
+  Caffe 2 depthwise 3x3 convolution operator
```

Comment:
nit: call it depthwise 3x3 winograd operator as we did at other places.

Reply:
Done.  Will upload a patch shortly.